### ISSUE TITLE
I would like to help update the katacoda CLI

### ISSUE THREAD

**Issue by managedkaos** [katacoda/cli#3]
### TLDR: I would like to assist in updating the Katacoda CLI.

Hello!  I develop content for students learning DevOps concepts and tools.  Particularly, I've developed content for LinkedIn Learning and Pluralsight.

I would like to use Katacoda to supplement lessons that can make use of preconfigured compute environments.  Honestly, it's perfectly suited to expedite learning by offering an environment that's ready to go.

After getting familiar with the authoring environment, I was able to develop some scenarios and a course in just a couple of hours.  I did all of that manually without using the CLI tool (even though I had it installed).

Now that I'm more familiar with the creation flow, I think I could move even faster using the CLI.  I'm already using the CLI in my repo for scenario validation.

I'd like to help update the CLI by remediating the following errors that are issued when the tool is installed:

```
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated source-map-resolve@0.[5]().3: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@2.0.3: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated @hapi/topo@3.1.[6](): This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/bourne@1.3.2: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated request-promise@4.2.6: request-promise has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated @hapi/address@2.1.4: Moved to 'npm install @sideway/address'
npm WARN deprecated @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated cli-ux@5.6.[7](h): Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm WARN deprecated cli-ux@5.3.1: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm WARN deprecated cli-ux@4.[9]().3: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm WARN deprecated cli-ux@4.9.3: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm WARN deprecated @hapi/joi@[15]().1.1: Switch to 'npm install joi'
```

Looking forward to helping!
